.env/.env.template

```diff
@@ -17,13 +17,13 @@ COOKIE_MAX_AGE=86400000
 REDIS_CONNECTION_STRING=
 
 # IDP HET ARCHIEF
-SAML_IDP_META_DATA_ENDPOINT=https://idp-tst.hetarchief.be/saml2/idp/metadata.php?output\=xml
+SAML_IDP_META_DATA_ENDPOINT=https://hetarchief-idp-int.do.viaa.be/saml2/idp/metadata.php?output\=xml
```

Comment:
Het kan wrs geen erg dat deze mee in de template staan? Deze worden meestal leeg gelaten maar het is wel een reference naar welke env vars er allemaal nodig zijn

Reply:
ok, verwijderd